recipes/griddataformats/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "GridDataFormats" %}
+{% set version = "0.3.3" %}
+{% set sha256 = "938f0efcb3bc2f58ec85048b933942da8a52c134170acc97cb095f09d3698fbd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - six
+    - numpy
+    - scipy
+
+  run:
+    - six
+    - numpy
+    - scipy
+
+test:
+  # Some package might need a `test/commands` key to check CLI.
+  imports:
+    - gridData
```

Comment:
I'm not sure why this is failing on some platforms. It almost looks like it's not attempting to construct the test environment. My only guess is that it's because python isn't a run time requirement, but that's just a shot in the dark.